docs/src/_includes/layouts/doc.html

```diff
@@ -102,6 +102,7 @@ <h1>{{ title }}</h1>
             {% include "partials/docs-footer.html" %}
         </div>
     </div>
+    <a class="scroll-up-btn" id="scroll_up_btn" href="#scroll_to_top">↑</a>
```

Comment:
We are using id for js and class for css can we use id for both please?

Reply:
and Im not sure if we are supposed to use up arrow icon like this. Can we rather replace it with svg? so that it doesn't break?